src/constants/networks.ts

```diff
@@ -37,6 +38,8 @@ export enum ChainId {
   OPTIMISM = 10,
 }
 
+export type ChainIdType = ChainId | string
```

Comment:
Change this to `export type ChainIdType = ChainId | 'allChain'`

Then change other places in code to use this. Examples:
 - https://github.com/KyberNetwork/promm-analytics/blob/danh/elastic-all-chain/src/components/Layout/SideNav.tsx#L241 `networkListToShow: ChainIdType[]`
 - https://github.com/KyberNetwork/promm-analytics/blob/danh/elastic-all-chain/src/components/Layout/SideNav.tsx#L266 `chainId: ChainIdType`
 - https://github.com/KyberNetwork/promm-analytics/blob/danh/elastic-all-chain/src/constants/index.ts#L7 `export const ALL_CHAIN_ID: ChainIdType = 'allChain'`
 - https://github.com/KyberNetwork/promm-analytics/blob/danh/elastic-all-chain/src/state/application/actions.ts#L22

Reply:
`export const SUPPORTED_NETWORKS = [...SHOW_NETWORKS, ALL_CHAIN_ID].filter((e) => e != ChainId.AURORA) as ChainIdType[]`
solve the error.